Dockerfile.erb

```diff
@@ -16,6 +16,14 @@ FROM alpine:3.6
 
 MAINTAINER GoCD <go-cd-dev@googlegroups.com>
 
+# Important: this values are yet read only, you cannot and should not change them, rather use them in your scripts
+ENV GOCD_DATA_HOME=/godata
+ENV GOCD_DB_HOME=/godata/db
+ENV GOCD_ADDONS_HOME=/godata/addons
+ENV GOCD_PLUGIN_HOME=/godata/plugins
+ENV GOCD_PLUGIN_EXTERNAL_HOME=/godata/plugins/external
+ENV GOCD_ARTIFACTS_HOME=/godata/artifacts
```

Comment:
Could you add another couple of vars for `/godata/config` `/godata/logs`? Also, if you could rollback the changes to `Dockerfile`, we can merge this PR.